pkgs/servers/roon-server/default.nix

```diff
@@ -0,0 +1,64 @@
+{ alsaLib, alsaUtils, cifs-utils, fetchurl, ffmpeg, libav, mono, stdenv }:
+
+stdenv.mkDerivation rec {
+  name = "roon-server";
+  version = "100600401";
+
+  src = fetchurl {
+    url = "http://download.roonlabs.com/updates/stable/RoonServer_linuxx64_${version}.tar.bz2";
+    sha256 = "121mmdh35q4bpgsqhcps6a6q1f4ld9v4hq9gp181bf2n779pk8sh";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    # Check script
+    sed -i '3i PATH=$PATH:${lib.makeBinPath [ cifs-utils ffmpeg libav ]}' check.sh
+    sed -i '/check_ulimit$/d' check.sh
+
+    # Start script
+    sed -i '3i PATH=$PATH:${lib.makeBinPath [ alsaUtils cifs-utils ffmpeg libav ]}' start.sh
```

Comment:
Last thing - we're getting eval errors. You need to either pass in ```lib``` or do ```stdenv.lib.makeBinPath``` here.

Reply:
Done.